.github/readmes/javascript/_setup-2.md

```diff
@@ -20,6 +20,27 @@ prisma2 lift up
 
 You can now use the [SQLite Browser](https://sqlitebrowser.org/) to view and edit your data in the `./prisma/dev.db` file that was created when you ran `prisma2 lift up`.
 
+<Details>
+<Summary><b>Alternative: </b>Connect to your own database</Summary>
+
+Prisma supports [other databases](https://github.com/prisma/prisma2/blob/master/docs/supported-databases.md). If you would like to connect to your own database, you can do so by specifying a different data source in the [Prisma schema file](prisma/schema.prisma):
+
+```
+datasource db {
+    provider = "DATABASE_PROVIDER"
+    url      = "DATABASE_PROVIDER://USERNAME:PASSWORD@HOST:PORT/DATABASE?schema=SCHEMA_FILE_NAME"
```

Comment:
The placeholder `SCHEMA_FILE_NAME` should probably just be `SCHEMA_NAME` since this isn't about a file but rather the name of the [PostgreSQL schema](https://www.postgresql.org/docs/9.1/ddl-schemas.html). Also, this URL pattern only applies to the PostgreSQL connector, for MySQL a `schema` is not part of the connection string :)

(PostgreSQL and MySQL have different semantics for the terms "schema" and "database", happy to quickly discuss this in a bit more detail with you if you're interested :)

Reply:
Yes, tell me more!